packages/cli/index.js

```diff
@@ -337,6 +337,13 @@ export async function useSpace(did) {
  */
 export async function spaceInfo(opts) {
   const client = await getClient()
+  if (typeof opts.space === 'boolean') {
+    console.error(
+      'Error: --space requires a DID value (e.g. --space did:key:...)'
+    )
+    process.exit(2)
+  }
+
   const spaceDID = opts.space ?? client.currentSpace()?.did()
```

Comment:
oh I see - it looks like the issue is that if --space doesn't have an argument it's true here rather than defaulting. I'd adjust this check to account for this - something like 
```suggestion
  // opts.space is set to `true` if --space is not specified - in this case we should call back to the DID of the current space
  const spaceDID = (!opts.space || (opts.space === true)) ? client.currentSpace()?.did() : opts.space
```